packages/twenty-front/src/modules/object-record/record-field/meta-types/display/components/RelationToOneFieldDisplay.tsx

```diff
@@ -9,7 +9,7 @@ export const RelationToOneFieldDisplay = () => {
   const { fieldValue, fieldDefinition, generateRecordChipData } =
     useRelationToOneFieldDisplay();
 
-  const { disableChipClick } = useContext(FieldContext);
+  const { disableChipClick, triggerEvent } = useContext(FieldContext);
```

Comment:
open-quesiton: do we really need to pass triggerEvent here, what's the usecase where we don't want it to be 'MOUSE_DOWN' regarding relationToOne or RecordChip?